.github/workflows/ci.yaml

```diff
@@ -37,9 +37,55 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
-      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491
+      - name: Check out platform
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
         with:
-          go-version-file: 'go.mod'
-          cache: false
-      - name: e2e Tests
-        run: echo "No e2e tests yet." && exit 0
+          repository: opentdf/platform
+          path: platform
+      - name: Set up go
+        uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7
+        with:
+          go-version-file: 'platform/service/go.mod'
+          check-latest: false
+          cache-dependency-path: |
+            platform/service/go.sum
+            platform/examples/go.sum
```

Comment:
I wonder if we can skip anything with the examples module since we don't depend on it in `otdfctl` or need to run anything against it?

Reply:
tru, ya we should only need service i think